### PR TITLE
PB 2 Created Add Delivery Employee sql file

### PR DESCRIPTION
**Pull request by amchugh689**
This updates the setup.sql file to allow creation of a Delivery Employee table. It also adds a sql file to allow addition of new employees to this table.

### CODE REVIEW

**Review by Samuel-BlankAmber — APPROVED** [review]
Brill

File: sql/setup.sql [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
\ No newline at end of file
+USE db_PICASO_OdhranH;
+
+# SQL for creating Delivery Employee table
+CREATE TABLE DeliveryEmployee(
+DEmpID int auto_increment not null PRIMARY KEY,
+DEmpForename varchar(20) not null,
+DEmpSurname varchar(20) not null,
+DEMPBankNum VARCHAR(16) not null,
+NationalInsuranceNum Varchar(9) not null
```
Comment (jkelso662) [inline sql/setup.sql]: Indentation here looks a bit naff!

File: sql/AddDeliveryEmp.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Change values in brackets
+#
+# In order to add multiple, copy line and change values
+# EG:
+# Insert into DeliveryEmployee(DEmpForename, DEmpSurname, DEmpBankNum, NationalInsuranceNum) VALUES
+# ('Forename','Surname','PhoneNum', 'NI Num'),
+# ('Forename','Surname','PhoneNum', 'NI Num');
+#
+Insert into DeliveryEmployee(DEmpForename, DEmpSurname, DEmpBankNum, NationalInsuranceNum) VALUES
+('Forename','Surname','PhoneNum', 'NI Num');
```
Comment (jkelso662) [inline sql/AddDeliveryEmp.sql]: What sort of person is Mr Forename Surname? :P Surely you need more data than just one entry? And 'NI Num' doesn't seem like a great example!

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Nearly there. Fix the indentation issue, add some test data and then good to go! Better merge request description but make sure you reference what backlog item your MR is for.

File: sql/AddDeliveryEmp.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Use this SQL file to add new Delivery Employees to the database.
```
Comment (jkelso662) [inline sql/AddDeliveryEmp.sql:1]: The naming of this file is completely different to that of merge request 1!

File: sql/AddDeliveryEmp.sql [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+('Rhiannon','Wheeler','733864730294', '83472342'),
+('Lacie','Abbot','827364518203','82938475'),
+('Milan','Keith', '987678543212', '091736543'),
+('Michelle','McDaniel', '271836479087','187326354'),
+('Santiago', 'McCann', '28194374637', '391837453'),
+('Luqman', 'Tapia', '3827362734', '123456789'),
+('Julia', 'Winter', '198273647987', '218767835'),
+('Pablo','Kelly', '736288374892', '63728193'),
+('Ridwan','Mata','987892374432', '123321234'),
+('Gerald', 'Cunningham', '837267378234', '38436238');
```
Comment (jkelso662) [inline sql/AddDeliveryEmp.sql]: Much better, although those don't really look like national insurance numbers!

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Nearly there. Make sure you discuss with your team what file name conventions you are using.

**Review by jkelso662 — APPROVED** [review]
Fab! Remember to reference what feature your MR is for in the descriptions.

**Pull request merged**